testing/test_streaming.py

```diff
@@ -148,36 +169,6 @@ def test_parallel_queued_parallelize_correctly():
     assert np.array_equal(outputs_streaming, outputs_vanilla)
 
 
-def test_parallel_queued_parallelize_correctly():
-    sleep_time = 0.001
-    p = SequentialQueuedPipeline([
-        ('1', 4, 10, Pipeline([ForEachDataInput(Sleep(sleep_time=sleep_time)), MultiplyByN(2)])),
```

Comment:
Okay for the test being removed because it was probably instable / flickering... but also, this test seemed to have good coverage and might be worth to be converted to two separate tests without a comparison of time between the two pipelines.

Reply:
it was a duplicate. the other copy is still there

Reply:
Ok, moved this to : 
https://github.com/Neuraxio/Neuraxle/issues/408